### PR TITLE
Release notes draft v1.24.0 rc.1

### PR DESCRIPTION
**Pull request by AuraSinis**
Major Themes added

Action items were mistreated and placed outside actionable items section, fixed manually

holding for a requested release note from @nate-double-u which I can't find

/hold

/cc @JamesLaverack

### CODE REVIEW

**Review by JamesLaverack — APPROVED** [review]
/lgtm

File: releases/release-1.24/release-notes/release-notes-draft.md [hunk 1]
```diff
@@ -1,16 +1,108 @@
-## Urgent Upgrade Notes 
+## Major Themes
+
+### Dockershim Removed from kubelet
+
+After its deprecation in v1.20, the dockershim component has been removed from the kubelet.
+From v1.24 onwards, you will need to either use one of the other [supported runtimes](https://kubernetes.io/docs/setup/production-environment/container-runtimes/) (such as containerd or CRI-O)
+or use cri-dockerd if you are relying on Docker Engine as your container runtime.
+For more information about ensuring your cluster is ready for this removal, please
+see [this guide](/blog/2022/03/31/ready-for-dockershim-removal/).
```
Comment (sftim) [inline releases/release-1.24/release-notes/release-notes-draft.md:9]: Shouldn't this be an absolute URL? eg
```suggestion
see [this guide](https://blog.k8s.io/2022/03/31/ready-for-dockershim-removal/).
```

File: releases/release-1.24/release-notes/release-notes-draft.md [hunk 2]
```diff
@@ -1,16 +1,108 @@
...
+
+With Kubernetes 1.24, the [gRPC probes functionality](https://github.com/kubernetes/enhancements/issues/2727)
+has entered beta and is available by default. You can now [configure startup, liveness, and readiness probes](/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes/#configure-probes) for your gRPC app
+natively within Kubernetes, without exposing an HTTP endpoint or
+using an extra executable.
+
+### Kubelet Credential Provider Graduates to Beta
+
+Originally released as Alpha in Kubernetes 1.20, the kubelet's support for
+[image credential providers](/docs/tasks/kubelet-credential-provider/kubelet-credential-provider/)
```
Comment (sftim) [inline releases/release-1.24/release-notes/release-notes-draft.md:59]: For the release notes, this should be an absolute URL:
```suggestion
[image credential providers](https://k8s.io/docs/tasks/kubelet-credential-provider/kubelet-credential-provider/)
```

File: releases/release-1.24/release-notes/release-notes-draft.md [hunk 3]
```diff
@@ -1,16 +1,108 @@
...
+
+### Contextual Logging in Alpha
+
+Kubernetes 1.24 has introduced [contextual logging](https://github.com/kubernetes/enhancements/issues/3077)
+that enables the caller of a function to control all aspects of logging (output formatting, verbosity, additional values and names).
+
+### Avoiding Collisions in IP allocation to Services
+
+Kubernetes 1.24 introduced a new opt-in feature that allows you to
+[soft-reserve a range for static IP address assignments](/docs/concepts/services-networking/service/#service-ip-static-sub-range)
```
Comment (sftim) [inline releases/release-1.24/release-notes/release-notes-draft.md:72]:
```suggestion
[soft-reserve a range for static IP address assignments](https://k8s.io/docs/concepts/services-networking/service/#service-ip-static-sub-range)
```

File: releases/release-1.24/release-notes/release-notes-draft.md [hunk 4]
```diff
@@ -1,16 +1,108 @@
...
+There is work under way to [migrate the internals of in-tree storage plugins](https://github.com/kubernetes/enhancements/issues/625) to call out to CSI Plugins,
+while maintaining the original API.
+The [Azure Disk](https://github.com/kubernetes/enhancements/issues/1490)
+and [OpenStack Cinder](https://github.com/kubernetes/enhancements/issues/1489) plugins
+have both been migrated.
+
+### gRPC Probes Graduate to Beta
+
+With Kubernetes 1.24, the [gRPC probes functionality](https://github.com/kubernetes/enhancements/issues/2727)
+has entered beta and is available by default. You can now [configure startup, liveness, and readiness probes](/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes/#configure-probes) for your gRPC app
```
Comment (sftim) [inline releases/release-1.24/release-notes/release-notes-draft.md:52]:
```suggestion
has entered beta and is available by default. You can now [configure startup, liveness, and readiness probes](https://k8s.io/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes/#configure-probes) for your gRPC app
```

**Review by sftim — COMMENTED** [review]
A few more post-merge comments.

**Pull request merged**